src/bitcoin_node_cli.hpp

```diff
@@ -32,10 +34,18 @@ class bitcoin_node_cli final {
 		 * @return new bitcoin address
 		 */
 		std::string get_new_address() const;
+		static bool curl_initialized;
```

Comment:
set this by default to =false; 
to remove any chance of this defaulting to true by mistake

in .cpp